testing/cloud_layouts/script.sh

```diff
@@ -341,18 +354,91 @@ function bootstrap_static() {
   terraform apply -auto-approve -no-color | tee "$cwd/terraform.log" || return $?
   popd
 
-  if ! master_ip="$(grep "master_ip_address_for_ssh" "$cwd/terraform.log"| cut -d "=" -f2 | tr -d " ")" ; then
+  if ! master_ip="$(grep "master_private_ip" "$cwd/terraform.log"| cut -d "=" -f2 | tr -d " ")" ; then
```

Comment:
Can we keep `master_ip_address_for_ssh` name here? My PR depends on this name

Reply:
done